modules/gitter/widget.go

```diff
@@ -102,11 +94,10 @@ func (widget *Widget) display() {
 		return
 	}
 
-	widget.View.SetWrap(true)
-	widget.View.Clear()
-	widget.View.SetTitle(widget.ContextualTitle(fmt.Sprintf("%s - %s", widget.CommonSettings.Title, widget.settings.roomURI)))
-	widget.View.SetText(widget.contentFrom(widget.messages))
-	widget.View.Highlight(strconv.Itoa(widget.selected)).ScrollToHighlight()
+	title := fmt.Sprintf("%s - %s", widget.CommonSettings.Title, widget.settings.roomURI)
+
+	widget.Redraw(title, widget.contentFrom(widget.messages), true)
+	//widget.View.Highlight(strconv.Itoa(widget.selected)).ScrollToHighlight()
```

Comment:
I don't have any widgets with highlight, so not 100% sure what is going on here and/or if this is a potential race condition that we should also try to abstract away